exercises/practice/perceptron/.meta/tests.toml

```diff
@@ -0,0 +1,25 @@
+# This is an auto-generated file.
```

Comment:
Is it? I don't see a perceptron exercise in problem-specifications.

Reply:
I'm not sure if I fixed this or not. I've deleted the auto-generated comment. I had to manually create the UUID since when I ran configlet, it kept saying everything was done even when there were no UUIDs in this file (sorry I can't remember the exact message). Let me know if I need to do something else

Reply:
Sorry, I'm not very familiar with what configlet needs when we're writing a practice exercise from scratch (one where there is not an upstream definition in problem-specifications). Someone in @exercism/reviewers might know.

Reply:
Looking at some problem-specifications, I'm wondering if psedorandom test generation is possible on the platform. Checking `canonical-data.json` for Alphametics shows each exercise with data such as a unique UUID, plus the input and expected result. If this file is necessary and needs the hardcoded input and expected output, although possible, it'll be a bit more work to produce.

Reply:
canonical-data isn't required. You can see parametric tests in the rational-numbers exercise and elsewhere.

Reply:
Great! I'd forgotten about the testset at the end of Rational Numbers :)